modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -0,0 +1,79 @@
+
+You must configure several options in your VMware Cloud VMC environment prior to installing {product-title} on VMware vSphere. Ensure your VMC environment has the following prerequisites:
+
+* Create a non-exclusive, DHCP-enabled, NSX-T network segment and subnet. Other virtual machines (VM) can be hosted on the subnet, but at least eight IP addresses must be available for the {product-title} deployment.
+
+ifdef::ipi[]
+* Allocate two IP addresses, outside the DHCP range, and configure them with reverse DNS records.
+** A DNS record for `api.<cluster_name>.<base_domain>` pointing to the allocated IP address.
+** A DNS record for `*.apps.<cluster_name>.<base_domain>` pointing to the allocated IP address.
+endif::ipi[]
+
+* Configure the following firewall rules:
+** An ANY:ANY firewall rule between the {product-title} compute network and the Internet. This is used by nodes and applications to download container images.
+** An ANY:ANY firewall rule between the installation host and the software-defined data center (SDDC) management network on port 443. This allows you to upload the {op-system-first} OVA during deployment.
+** An HTTPS firewall rule between the {product-title} compute network and vCenter. This connection allows {product-title} to communicate with vCenter for provisioning and managing nodes, persistent volume claims (PVC), and other resources.
+
+* You must have the following information to deploy {product-title}:
+** The {product-title} cluster name, such as `vmc-prod-1`.
+** The base DNS name, such as `companyname.com`.
+** If not using the default, the pod network CIDR and services network CIDR must be identified, which are set by default to `10.128.0.0/14` and `172.30.0.0/16`, respectively. These CIDRs are used for pod-to-pod and pod-to-service communication and are not accessible externally; however, they must not overlap with existing subnets in your organization.
+** The following vCenter information:
+*** vCenter hostname, username, and password
+*** Datacenter name, such as `SDDC-Datacenter`
+*** Cluster name, such as `Cluster-1`
+*** Network name
+*** Datastore name, such as `WorkloadDatastore`
++
+[NOTE]
+====
+It is recommended to move your vSphere cluster to the VMC `Compute-ResourcePool` resource pool after your cluster installation is finished.
+====
+
+* A Linux-based host deployed to VMC as a bastion.
+** The bastion host can be {op-system-base-full} or any another Linux-based host; it must have Internet connectivity and the ability to upload an OVA to the ESXi hosts.
+** Download and install the OCP CLI tools to the bastion host.
+*** The `openshift-install` installation program
+*** The OpenShift CLI (`oc`) tool
+
+[NOTE]
+====
+You cannot use the VMware NSX Container Plugin for Kubernetes (NCP) and NSX is not used as the OpenShift SDN. The version of NSX currently available with VMC is incompatible with the version of NCP certified with {product-title}.
```

Comment:
This first sentence sounds weird to me - might just be a lack of knowledge on the subjects, but just take another look to make sure it's what you meant.